vaadin-date-picker-flow-parent/vaadin-date-picker-flow-integration-tests/test/date-picker-connector.test.ts

```diff
@@ -0,0 +1,58 @@
+import { expect, fixtureSync } from '@open-wc/testing';
+import { init, extractDateParts, datepickerConnector, type FlowDatePicker } from './shared.js';
+
+describe('date-picker connector', () => {
+  let datePicker: FlowDatePicker;
+
+  beforeEach(() => {
+    datePicker = fixtureSync('<vaadin-date-picker></vaadin-date-picker>');
+    init(datePicker);
+  });
+
+  it('should not reinitialize the connector', () => {
+    const connector = datePicker.$connector;
+    datepickerConnector.initLazy(datePicker);
+    expect(datePicker.$connector).to.equal(connector);
+  });
+
+  const DATE = new Date();
```

Comment:
Would it be more reliable to use an explicit date?

Reply:
Makes sense. Done.